package.json

```diff
@@ -33,59 +28,67 @@
     "build:css": "npm-run-all build:css:sass build:css:bannerize",
     "build:css:bannerize": "bannerize dist/videojs-overlay.css --banner=scripts/banner.ejs",
     "build:css:sass": "node-sass src/plugin.scss dist/videojs-overlay.css --output-style=compressed --linefeed=lf",
-    "build:js": "npm-run-all build:js:babel build:js:browserify build:js:bannerize build:js:uglify",
-    "build:js:babel": "babel src -d es5",
+    "build:js": "npm-run-all build:js:rollup-modules build:js:rollup-umd build:js:bannerize build:js:uglify",
     "build:js:bannerize": "bannerize dist/videojs-overlay.js --banner=scripts/banner.ejs",
-    "build:js:browserify": "browserify . -s videojs-overlay -g browserify-shim -t browserify-versionify -o dist/videojs-overlay.js",
-    "build:js:uglify": "uglifyjs dist/videojs-overlay.js --comments --mangle --compress -o dist/videojs-overlay.min.js",
-    "build:test": "babel-node scripts/build-test.js",
-    "build:test:browserify": "browserify `find test -name '*.test.js'` -t babelify -o dist-test/videojs-overlay.js",
-    "change": "chg add",
-    "clean": "rimraf dist test/dist es5 && mkdirp dist test/dist es5",
+    "build:js:rollup-modules": "rollup -c scripts/modules.rollup.config.js",
+    "build:js:rollup-umd": "rollup -c scripts/umd.rollup.config.js",
+    "build:js:uglify": "uglifyjs dist/videojs-overlay.js --comments --mangle --compress  -o dist/videojs-overlay.min.js",
+    "build:test": "rollup -c scripts/test.rollup.config.js",
+    "clean": "rimraf dist test/dist",
+    "postclean": "mkdirp dist test/dist",
     "lint": "vjsstandard",
-    "start": "babel-node scripts/server.js",
+    "prestart": "npm run build",
```

Comment:
FWIW, it was removed from the generator because with major versions transitioning to and from Spellbook, it became really brittle and hard to maintain for too little benefit, I think.